Merge from Ubuntu, ensure that the /lib/modules/<version>/updates/dkms
directory is removed if it is empty after a removal.  This allows the
modules directory to be removed as it becomes empty.
--- a/kernel_prerm.d_dkms
+++ b/kernel_prerm.d_dkms
@@ -13,4 +13,7 @@
 done < <(dkms status -k $inst_kern 2>/dev/null | grep ": installed")
 fi
 
+rmdir --ignore-fail-on-non-empty \
+	"/lib/modules/$inst_kern/updates/dkms" \
+	"/lib/modules/$inst_kern/updates"
 exit 0
